tests/lib/testers/rule-tester.js

```diff
@@ -13,7 +13,7 @@
 const sinon = require("sinon"),
     EventEmitter = require("events"),
     RuleTester = require("../../../lib/testers/rule-tester"),
-    assert = require("chai").assert;
+    assert = require("assert");
```

Comment:
Can we keep using `chai.assert` here? I think we we generally use `chai.assert` in tests. (We use the regular `assert` module in `rule-tester` to avoid needing to add a dependency.) `chai.assert` has a few advantages over `assert`, e.g. `chai.assert.throws(func, "error string")` tests against the error message as expected rather than creating a message for the assertion error.

Reply:
Sure, I will change it back. I personally would love to change the `assert` implementation to also check for the error message but that would be probably to much of a breaking change.